wrangler-api/src/main/java/io/cdap/wrangler/api/Directive.java

```diff
@@ -115,4 +116,13 @@
    * @see io.cdap.wrangler.api.parser.TokenType
    */
   UsageDefinition define();
+
```

Comment:
Should not be added to this interface. Could we add another interface named Mutation that has this method. Only directives that implement that interface will that. This is very clunky.

Reply:
Addressed in https://github.com/data-integrations/wrangler/pull/365